core/src/test/java/org/springframework/security/authentication/DefaultAuthenticationEventPublisherTests.java

```diff
@@ -137,6 +138,49 @@ public void unknownFailureExceptionIsIgnored() {
 		verifyZeroInteractions(appPublisher);
 	}
 
+	@Test(expected = IllegalArgumentException.class)
```

Comment:
Spring Security prefers assertJ. Would you mind changing these from using the `excepted` attribute in the annotation to using `assertThatThrownBy`?

e.g.

```java
// ...
assertThatThrownBy(() -> publisher.setAdditionalExceptionMappings(mappings))
        .isInstanceOf(IllegalArgumentException.class);
```